pkg/comp-functions/functions/common/postgresql.go

```diff
@@ -19,29 +17,61 @@ const (
 	PgSecretName         = "pg-creds"
 )
 
-func AddPostgreSQL(svc *runtime.ServiceRuntime, comp InfoGetter, psqlParams *vshnv1.VSHNPostgreSQLParameters, pgBouncerConfig map[string]string) error {
+type PostgreSQLDependencyBuilder struct {
```

Comment:
I'm having issues with the naming of this type and the function below. But I don't really have a better name in mind right now

Reply:
I also have no idea what would be the best name :)